src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionPriorityProvider.cs

```diff
@@ -12,16 +13,36 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.Suggestions;
 /// Set of de-prioritized analyzers that were moved down from 'Normal' to 'Low' priority bucket. Note that this set is
 /// owned by the <see cref="SuggestedActionsSourceProvider.SuggestedActionsSource"/> and shared across priority buckets.
 /// </param>
+/// <param name="lowPriorityAnalyzerSupportedDiagnosticIds">
+/// Combined set of supported diagnostic ids from all de-prioritized analyzers
+/// </param>
 internal sealed class SuggestedActionPriorityProvider(
     CodeActionRequestPriority priority,
-    ConcurrentSet<DiagnosticAnalyzer> lowPriorityAnalyzers)
+    ConcurrentSet<DiagnosticAnalyzer> lowPriorityAnalyzers,
+    ConcurrentSet<string> lowPriorityAnalyzerSupportedDiagnosticIds)
```

Comment:
feeling like i want these two sets merged into one type.  don't feel strongly about it.

Reply:
Changed to a struct that contains both ConcurrentSets.